src/components/nameBadge/nameBadge.tsx

```diff
@@ -3,10 +3,18 @@ import S from '@/components/nameBadge/nameBadge.module.css';
 interface NameBadgeProps {
   color: 'yellow' | 'orange' | 'green' | 'blue' | 'brown' | 'pink';
   letter: string;
+  layer?: 'layer' | null;
+  index?: number | undefined;
```

Comment:
```suggestion
  layer?: 'layer';
  index?: number;
```

이렇게 해도 적용될 것 같은데 맞나요 ?!?

Reply:
아하! 수정하겠습니다!